### PR TITLE
Update READ.me

### PR DESCRIPTION
**Pull request by cookiethecatkr**
Fixed typo. Re-arranged items.
Removed the broken link.

### CODE REVIEW

File: 7-bank-project/README.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ In this project, you'll learn how to build a fictional bank. These lessons inclu
 
 These lessons were written with :hearts: by [Yohan Lasorsa](https://twitter.com/sinedied).
 
-If you're interested to learn how to build the [server API](./api/README) used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
+If you're interested to learn how to build the server API used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
```
Comment (jlooper) [inline 7-bank-project/README.md:19]: Is the link broken? Can we fix it rather than remove it?

**Review by jlooper — CHANGES_REQUESTED** [review]

File: 7-bank-project/README.md (hunk 1 above)
Comment (cookiethecatkr) [reply 7-bank-project/README.md]: Good idea!  Today's my first day of contribution. Please let me know if I am missing anything. :)

**Review by cookiethecatkr — COMMENTED** [review]
I've fixed the link.

File: 7-bank-project/README.md [hunk 2]
```diff
@@ -16,6 +16,6 @@ In this project, you'll learn how to build a fictional bank. These lessons inclu
 
 These lessons were written with :hearts: by [Yohan Lasorsa](https://twitter.com/sinedied).
 
-If you're interested to learn how to build the [server API](./api/README) used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
+If you're interested to learn how to build the [server API](https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/7-bank-project/api/README.md) used in these lessons, you can follow [this series of videos](https://aka.ms/NodeBeginner) (in particular videos 17 through 21).
```
Comment (jlooper) [inline 7-bank-project/README.md:19]: I think this can be a relative link

**Review by jlooper — CHANGES_REQUESTED** [review]

File: 7-bank-project/README.md (hunk 2 above)
Comment (cookiethecatkr) [reply 7-bank-project/README.md]: I agree. Thank you for your feedback. I've changed it.

Comment (cookiethecatkr) [reply 7-bank-project/README.md]: @jlooper  Do I click the "Resolve conversation"?  I'm new, so I don't know the common protocol. :(

Comment (jlooper) [reply 7-bank-project/README.md]: This looks good, I'll merge! thank you

**Comment by sakkiiit** [comment]
> Fixed typo. Re-arranged items.
> 
> Removed the broken link.

**Pull request merged**